code/modules/mob/living/simple_animal/hostile/jungle/seedling.dm

```diff
@@ -30,6 +30,7 @@
 	robust_searching = TRUE
 	stat_attack = HARD_CRIT
 	move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	gold_core_spawnable = HOSTILE_SPAWN
```

Comment:
This is both unfair and unfun to fight against unless you have TK and a very good melee weapon; probably not a good idea.